website/static/js/search.js

```diff
@@ -394,9 +394,13 @@ var ViewModel = function(params) {
             var nullLicenseCount = data.aggs.total || 0;
             if ((data.aggs || {}).licenses)  {
                 $.each(data.aggs.licenses, function(key, value) {
-                    licenseCounts.filter(function(l) {
-                        return eqInsensitive(l.id, key);
-                    })[0].count(value);
```

Comment:
Rather than changing this to a `for` loop, you can just add a check after the filter:

```js
var licenseCount = licenseCounts.filter(function(l) {
    return eqInsensitive(l.id, key);
})[0];
if (licenseCount) {
    licenseCount.count(value);
}
```

Reply:
I thought about this, but doesn't filter go through the whole array? given that we just need to find one key, I thought breaking after finding it would make things faster, that we are not doing n^2 for every element. I guess it does not really matter since this is small list.

Reply:
Yeah, in this case we're dealing with such a small list that it doesn't really matter, and I find the `.filter` slightly more readable. I don't feel strongly either way. Follow your ❤️